addons/binding/org.openhab.binding.tplinksmarthome.test/src/test/java/org/openhab/binding/tplinksmarthome/internal/TPLinkSmartHomeDiscoveryServiceTest.java

```diff
@@ -35,8 +40,12 @@
  *
  * @author Hilbrand Bouwkamp - Initial contribution
  */
+@RunWith(value = Parameterized.class)
 public class TPLinkSmartHomeDiscoveryServiceTest {
 
+    private static final List<Object[]> TESTS = Arrays.asList(
+            new Object[][] { { "bulb_get_sysinfo_response_on", 11 }, { "rangeextender_get_sysinfo_response", 11 } });
```

Comment:
Does the var-args way of calling `Arrays.asList` not work here?

Reply:
Yes, but since it's a list of lists, it would be: ` Arrays.asList(new Object[] { "bulb_get_sysinfo_response_on", 11 }, new Object[]  { "rangeextender_get_sysinfo_response", 11 })`. I'm not sure if that would be an improvement.